build-system/tasks/size.js

```diff
@@ -14,35 +14,31 @@
  * limitations under the License.
  */
 
-var table = require('text-table');
 var del = require('del');
```

Comment:
Just one quick note - JSON was a suggestion. If you find it easier to do a text table - I'm all for it :)